index.js

```diff
@@ -1,61 +1,77 @@
-let books = [];
+const listOfBooks = document.querySelector('.list');
+const formBtn = document.querySelector('.form_button');
 
 const handleStorage = (data) => {
   localStorage.setItem('books', JSON.stringify(data));
 };
 
-const listOfBooks = document.querySelector('.list');
-const formBtn = document.querySelector('.form_button');
+class BookShelf {
+  constructor() {
+    this.books = [];
+  }
 
-function addBook(book) {
-  const newBook = { id: Math.random(), ...book };
-  const newArray = books.concat([newBook]);
-  handleStorage(newArray);
-}
+  addBook(book) {
+    const newBook = { id: Math.random(), ...book };
+    const newArray = this.books.concat([newBook]);
+    handleStorage(newArray);
+  }
 
-function removeBook(id) {
-  const deletedBook = document.getElementById(id);
-  books = books.filter((book) => book.id !== id);
-  deletedBook.parentElement.remove();
-  handleStorage(books);
+  removeBook(id) {
+    this.books = this.books.filter((book) => book.id !== id);
```

Comment:
- well done for using `concat` and `filter`. 
- Well done 👍 

![Capture](https://user-images.githubusercontent.com/63972670/213496427-20dd2427-a27f-49cd-9a76-90ea6606bb6c.PNG)